src/components/Editor/ConditionalPanel.js

```diff
@@ -22,6 +24,10 @@ function renderConditionalPanel({ condition, closePanel, setBreakpoint }:
   ReactDOM.render(
     dom.div(
       { className: "conditional-breakpoint-panel" },
+      dom.div({
+        className: "prompt",
+        dangerouslySetInnerHTML: { __html: "&raquo;" }
```

Comment:
yup, this is how we do it in another spot:
https://github.com/devtools-html/debugger.html/blob/master/src/components/shared/Dropdown.js#L42